client/src/components/SandwichShow.js

```diff
@@ -33,6 +36,50 @@ const SandwichShow = (props) => {
     getSandwich()
   }, [])
 
+  const newReview = () => {
+    if(!props.user){
+      alert("You must be logged in to make a review!")
```

Comment:
Im going to ask we remove alerts, as they manage state outside of a react app. If you want to show this, then make it rendered via state change. Otherwise, dont allow the form to be shown unless a user is signed in